src/drive/web/modules/views/OnlyOffice/Title.jsx

```diff
@@ -35,7 +36,7 @@ const Title = () => {
   )
 
   return (
-    <>
+    <div style={{ zIndex: '1' }}>
```

Comment:
Can you use one of the UI's var for zIndex? https://github.com/cozy/cozy-ui/blob/master/stylus/settings/z-index.styl#L55

Reply:
La valeur n'est pas importante en réalité puisque le container qui passe dessous n'a pas de z-index. Mais pour que ce soit plus clair, j'ai changé en `--zIndex-nav`